.github/workflows/ci.yml

```
@@ -370,6 +370,71 @@ jobs:
           ## Getting permission denied for docker volumes (e.g. postgres data)
           ## https://github.com/actions/upload-artifact/issues/192
 
+  ensure-system-types-were-generated:
+    name: Ensure system types were generated
+    runs-on: ubuntu-latest
+    needs: [setup, build-hash-graph]
+    steps:
+      - uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4.0.0
+
+      - uses: actions/cache@704facf57e6136b1bc63b828d79edcd491f0ee84 # v3.3.2
+        with:
+          path: apps/hash-frontend/.next/cache
+          key: ${{ runner.os }}-frontend-next-cache-${{ hashFiles('yarn.lock') }}-${{ hashFiles('apps/hash-frontend/**') }}
+          restore-keys: |
+            ${{ runner.os }}-frontend-next-cache-${{ hashFiles('yarn.lock') }}-
+            ${{ runner.os }}-frontend-next-cache
+
+      - uses: ./.github/actions/warm-up-repo
+
+      - name: Create temp files and folders
+        run: mkdir -p var/logs
+
+      - name: Load Docker images
+        uses: ./.github/actions/load-docker-images
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          hash-graph: true
+          hash-ai-worker-ts: true
+          hash-ai-worker-py: true
+          hash-integration-worker: true
+
+      - name: Launch external services
+        run: |
+          yarn turbo codegen --filter '@apps/hash-external-services'
+          # if we use turbo here, this would cost time with the docker build step needing to check that all the layers are cached
+          yarn workspace @apps/hash-external-services deploy up --detach
+
+      - name: Launch backend
+        run: |
+          yarn dev:backend 2>&1 | tee var/logs/backend.log & ## ampersand enables background mode
+          yarn wait-on --timeout 120000 http://0.0.0.0:5001
+
+      - name: Build and launch frontend
+        run: |
+          yarn workspace @apps/hash-frontend next build
+          yarn workspace @apps/hash-frontend start 2>&1 | tee var/logs/frontend.log & ## ampersand enables background mode
+          yarn wait-on --timeout 30000 http://0.0.0.0:3000
+
+      - name: Generate system types
+        env:
+          FRONTEND_URL: http://0.0.0.0:3000
```

Did you try localhost instead?
```suggestion
          FRONTEND_URL: http://127.0.0.1:3000
```

---

Unfortunately `http://127.0.0.1:3000` doesn't seem to have worked either (see [this run](https://github.com/hashintel/hash/actions/runs/6237832507/job/16932377540?pr=3199))